docs/differences_with_venv.rst

```diff
@@ -0,0 +1,17 @@
+Differences with venv
+=====================
+
+- ``venv`` is bundled with Python itself (starting from 3.3+), ``virtualenv``
```

Comment:
You removed the performance entry which is the main reason one would pick this.

Reply:
It defines no reason for said performance.

And that performance only comes into play if someone uses the `app-seed` feature, right?

Reply:
It's what most visible to end-users though and is on by default.